### PR TITLE
Change to `WORKDIR` before running commands in cloud project experiments.

### PR DESCRIPTION
**Pull request by Alan32Liu**
Mitigates the known issue where we don't automatically change to the `WORKDIR` defined in `Dockerfile` when running cloud experiments.

Question:
Would it be preferred if I introduce a flag for this?
(e.g., `--use_workdir` or `--workdir=/src/<project>`)
While this gives more flexibility, I feel `cd` to `WORKDIR` should always be preferred if we want the cloud experiments to behave the same as local ones.

### CODE REVIEW

File: infra/build/functions/project_experiment.py [hunk 1]
```diff
@@ -17,12 +17,17 @@
...
+import os
 import sys
 
-import google.auth
-
 import build_lib
 import build_project
+import google.auth
+
+sys.path.append(
```
Comment (oliverchang) [inline infra/build/functions/project_experiment.py:27]: This is a little hacky. We can avoid this by instead using the implementation here: https://github.com/google/oss-fuzz/blob/b5205aebc4caa161f16a98496dda820a02c83f1e/infra/build/functions/build_project.py#L211

**Review by oliverchang — APPROVED** [review]
Don't think we need a flag for this. This seems like a reasonable default that matches our prod infra.

File: infra/build/functions/project_experiment.py (hunk 1 above)
Comment (arezooalizadeh06) [reply infra/build/functions/project_experiment.py]: 🙏

**Comment by Alan32Liu** [comment]
Results are generated here:
https://pantheon.corp.google.com/storage/browser/oss-fuzz-experiment/_project_asts?project=oss-fuzz&pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&prefix=&forceOnObjectsSortingFiltering=false

Would it be ok to merge? @jonathanmetzman 
Thanks :)

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**